src/infill/SubDivCube.cpp

```diff
@@ -37,7 +37,7 @@ SubDivCube::~SubDivCube()
 void SubDivCube::precomputeOctree(SliceMeshStorage& mesh)
 {
     radius_addition = mesh.settings.get<coord_t>("sub_div_rad_add");
-    AngleRadians infill_angle = M_PI / 4.0;
+    AngleRadians infill_angle = 45;
```

Comment:
This still says `AngleRadians`, so the 45 gets wrapped around 2*pi using clock arithmetic. It then ends up at 1.01, which is about 58 degrees. Not what you wanted, I think. But you're right that `PointMatrix` accepts the angle in degrees. You just need to make the type an `AngleDegrees`.

Reply:
Yes, indeed!